requirements.txt

```diff
@@ -5,7 +5,7 @@ boto3>=1.9.96
 apache-libcloud>=0.20.1
 click>=7.0
 yamlordereddictloader>=0.4.0
-ansible>=2.7.1
+ansible>=2.7.1,<2.7.10
```

Comment:
??? 2.7.10 ?? is there any bug related to 2.7.10 could you report it on GitHub so that we can resolve it.

Reply:
I don't think it's a bug, but an API change.  It looks like two things were changed that impact us, neither of which I think is a bug.

1. you can no longer set variable_manager.extra_vars.  I changed this to `variable_manager._extra_vars = ......` but I think that's a bit hacky since `_extra_vars` is supposed to be private.
2. The `__init__` function for the `PlaybookExecutor` class no longer includes an options parameter.  I'm still working through how to get those loaded in the instance but I decided I'd rather take the short fix to get this PR submitted for now